OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,511 @@
+*   Vulnerable to Grieving attacks.
+*   Generally, these solutions focus on the fulfilment of the delivery, more so than measuring the speed and rate of the transfer.
+
+
+#### Optimistic Fair Exchange
+
+These solutions build on top of the Fair Exchange ones and adopt an Optimistic approach where some sort of Reputation or Stake is used to give both parties trust that the exchange will occur correctly. If one misbehaves, they can still rely on a dispute system.
+
+- [OptiSwap: Fast Optimistic Fair Exchange](INSERT URL)
+- [Asynchronous Protocols for Optimistic Fair Exchange](INSERT URL)
+- [Optimistic Fair Exchange with Multiple Arbiters](INSERT URL)
+
+**Known shortcomings of these approaches:**
+*   Setup + Dispute resolution time is non insignificant and non-trivial
+
+#### Reputation based
+
+Reputation based solutions give the possibility to speed up the transfer and improve the quality of the service, by leveraging the trust built over time between clients & providers.
+
+- [Proof of Delivery in a Trustless Network](INSERT URL)
+- [Proof-of-Prestige: A Useful Work Reward System for Unverifiable Tasks](INSERT URL)
+- [Mechanisms for Outsourcing Computation via a Decentralized Market](INSERT URL)
+- Other Gradual Fair Exchange Class of constructions
+
+**Known shortcomings of these approaches:**
+*   Not a direct shortcoming, but a challenge is that each participant my have different levels of sensitivity with regards to how much they are willing to rely on trust vs. strict verifiability. 
+
+#### Privacy focused
+
+There are multiple solutions with the goal of delivering readers privacy and writers privacy. These incur an additional cost in setup and/or latency in favor of the additional privacy preserving benefits. Disclaimer: At the time of writing, we haven't dived as deep into these set of solutions.
+
+- [Blockchain based Privacy-Preserving Software Updates with Proof-of-Delivery for Internet of Things](INSERT URL)
+- [SilentDelivery: Practical Timed-delivery of Private Information using Smart Contracts](INSERT URL)
+- [Bulletproofs+: Shorter Proofs for Privacy-Enhanced Distributed Ledger](INSERT URL)
+
+#### Traditional approaches (close to Web 2.0 world)
+
+
+These approaches suggest novel ways to add integrity checks to data transferred or to promote new ways for content distribution, but they don’t depart from the Web 2.0 world of assumptions such as trusted third parties and central points of control.
+
+- [The multimedia blockchain: a distributed and tamper-proof media transaction framework](INSERT URL)
+- [Reliable Client Accounting for P2P-Infrastructure Hybrids](INSERT URL)
+
+**Known shortcomings of these approaches:**
+*   Not fully aware of the challenges of building a decentralized network
+
+### Known attacks to be mitigated
+
+Here we list the attacks to consider when designing a solution. These are:
+
+*   Malicious actor forces provider to spend bandwidth without issuing payment (also known as Grieving Attack)
```

Comment:
```suggestion
*   Malicious actor forces provider to spend bandwidth without issuing payment (also known as Griefing Attack)
```
I don't appreciate the corruption either but it really is grief/griefing attack 🤷